NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -120,6 +121,50 @@ public ActionQuery()
                     };
                     return Codec.Encode(action.PlainValue);
                 });
+            Field<ByteStringType>(
+                name: "transferAsset",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "Address of sender.",
+                        Name = "sender",
+                    },
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "Address of recipient.",
+                        Name = "recipient",
+                    },
+                    new QueryArgument<NonNullGraphType<StringGraphType>>
+                    {
+                        Description = "A string value to be transferred.",
+                        Name = "amount",
+                    },
+                    new QueryArgument<NonNullGraphType<CurrencyType>>
+                    {
+                        Description = "A currency value to be transferred.",
```

Comment:
I'm not sure.. how about `type` instead of `value`.
```suggestion
                        Description = "A currency type to be transferred.",
```